src/backend/optimizer/plan/orca.c

```diff
@@ -194,12 +188,6 @@ optimize_query(Query *parse, ParamListInfo boundParams)
 	 * Fix ShareInputScans for EXPLAIN, like in standard_planner(). For all
 	 * subplans first, and then for the main plan tree.
```

Comment:
"For all subplans first", should refine the comment here:
please help to add more details of why there is no need to deal with the subplan.

Reply:
```suggestion
	 * Fix ShareInputScans for EXPLAIN, like in standard_planner(). Subplans
	 * will be visited from references to them during walking through the main
	 * plan tree(see shareinput_walker comments)
```

@interma, does such comment describe the reason in sufficient detail?

Reply:
ok, no problem, please add them.